spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -529,9 +533,7 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 								return;
 							}
 						}
-						if (ListenerConsumer.this.logger.isDebugEnabled()) {
-							ListenerConsumer.this.logger.debug("Committing on assignment: " + offsets);
-						}
+						ListenerConsumer.this.commitLogger.log("Committing on assignment: " + offsets);
```

Comment:
That's not good. We still have to check `isEnabled()` before concatenating strings.
Or what am I missing when we do that?

Reply:
doh - you are correct - will rework